dir.props

```diff
@@ -196,6 +196,10 @@
     <TargetGroup Condition="'$(TargetGroup)'=='' and $(Configuration.Contains('wpa81'))">wpa81</TargetGroup>
     <TargetGroup Condition="'$(TargetGroup)'=='' and $(Configuration.Contains('uap101aot'))">uap101aot</TargetGroup>
     <TargetGroup Condition="'$(TargetGroup)'=='' and $(Configuration.Contains('uap101'))">uap101</TargetGroup>
+
+    <ProjectRunConfig Condition="'$(TargetGroup)'!=''">$(OSGroup).$(Platform).$(TargetGroup).$(ConfigurationGroup).$(TestTFM)</ProjectRunConfig>
```

Comment:
you could simply not special case it and if targetgroup was "" it would just show up as double dots.